src/service.py

```diff
@@ -88,7 +88,9 @@ def _render_file(self, path: Path, content: str, mode: int) -> None:
     def is_active(self) -> bool:
         """Indicate if the jenkins agent service is active."""
         try:
-            return systemd.service_running(AGENT_SERVICE_NAME)
+            return os.path.exists(str(AGENT_READY_PATH)) and systemd.service_running(
```

Comment:
I think the `PATH.exists()` can be used here :D

Reply:
Thanks, I'll put this in a follow-up PR